decred/decred/util/helpers.py

```diff
@@ -214,12 +214,13 @@ def makeWebsocketURL(baseURL, path):
     Turn the HTTP/HTTPS URL into a WS/WSS one.
 
     Args:
-        url (str): The base URL.
+        baseURL (str): The base URL.
         path (str): The websocket endpoint path. e.g. path of "ws" will yield
             URL wss://yourhost.com/ws.
     Returns:
         string: the WebSocket URL.
     """
+    baseURL = f"wss://{baseURL}" if "//" not in baseURL else baseURL
     url = urlsplit(baseURL)
     scheme = "wss" if url.scheme in ("https", "wss") else "ws"
     return urlunsplit((scheme, url.netloc, f"/{path}", "", ""))
```

Comment:
Considered adjusting the `makeWebsocketURL` function to not rely on the `scheme` to determine whether to use `wss` or `ws` but there are a bunch of other places calling this function that rely on that behavior, so it seemed best to leave it as-is for now